packages/common/package.json

```diff
@@ -36,15 +36,23 @@
   },
   "dependencies": {
     "@qiwi/substrate": "^1.20.9",
-    "tslib": "^2.3.0",
-    "lodash": "^4.17.21",
-    "@types/lodash": "^4.14.170",
-    "xregexp": "^5.0.2",
     "@types/xregexp": "^4.3.0",
-    "flattie": "^1.1.0"
+    "flattie": "^1.1.0",
+    "lodash.clonedeep": "^4.5.0",
+    "lodash.get": "^4.4.2",
+    "lodash.invert": "^4.3.0",
+    "lodash.set": "^4.3.2",
+    "lodash.unset": "^4.5.2",
```

Comment:
is this need for common? Seems like it deps of `packages/schema`

Reply:
lodash was installed in common

Reply:
But these methods are not required, right?